### ISSUE TITLE
Share the train script?

### ISSUE THREAD

**Issue by Hansry** [atapour/monocularDepth-Inference#4]
Hello, Atapour. Thanks for the outstanding work. Can you share the training script? I am not very clearly understand the Loss function (3.1.1). I will appreciate it if you can reply. Thank you very much.

**Comment by atapour** [comment]
unfortunately, the training script is no longer available. However, as mentioned in the readme file, the code was based on the modified version of the code here: https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix

The loss function in part 3.1.1 is a simple L1 loss (https://pytorch.org/docs/stable/nn.html?highlight=loss#torch.nn.L1Loss) and a conditional GAN loss (https://github.com/pytorch/examples/tree/master/dcgan).
